package/base-files/Makefile

```diff
@@ -12,7 +12,7 @@ include $(INCLUDE_DIR)/version.mk
 include $(INCLUDE_DIR)/feeds.mk
 
 PKG_NAME:=base-files
-PKG_RELEASE:=205
+PKG_RELEASE:=206
```

Comment:
Why this cosmetic changes need package bump?

Reply:
As I understood it, this number must always be increased if something has changed.

Reply:
There are no functional changes, right? So I think, that the PKG version bump is not necessary. If you bump package version, then people are likely going to update this package, wasting flash erase cycles for nothing.

Reply:
> There are no functional changes, right?

yes

> if you bump package version, then people are likely going to update this package, wasting flash erase cycles for nothing.

Good to know.

Thanks for your explanation